snap/implicit.go

```diff
@@ -34,11 +34,13 @@ var implicitSlots = []string{
 	"hardware-observe",
 	"locale-control",
 	"log-observe",
+	"mir",
```

Comment:
I *think* mir should be in implicitClassicSlots, no? On an all snaps system mir is provided as a snap (therefore not needed in implicitSlots) but on a classic system (eg, one with unity8) then mir would be part of the system (that said, there are I think changes that are needed to make this interface work on classic for the unix label rules, but those can be fixed in a later PR). If this is only for kiosks on all-snaps systems (or at least this PR is), perhaps don't have mir be an implicit slot at all.